ocean_lib/common/ddo/ddo.py

```diff
@@ -242,7 +248,7 @@ def _read_dict(self, dictionary):
 
         self._other_values = values
 
-    def add_proof(self, checksums, publisher_account):
+    def add_proof(self, checksums: dict, publisher_account: object) -> None:
```

Comment:
I'm assuming the `publisher_account` is an instance of `eth_account.account.Account`. If I'm right, change `object` to `Account`